### ISSUE TITLE
Problems with the environment

### ISSUE THREAD

**Issue by X1AOX1A** [bperezorozco/ordinal_tsf#4]
It's hard to configure the environment when I'm running code. Could you please provide the requirement.txt file to help me configure the environment?

Thx!

**Comment by solalatus** [comment]
Yes, please, do so! I struggled a lot to guess the exact versions, and it is nearly impossible! I would love to reproduce!
